Solution/HW06.Task1/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+namespace HW06.Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] data1 = new int[10];
+            int[] data2 = new int[10];
+            int[] final_data = new int[10];
+            Random rnd = new Random();
+            for (int x = 0; x < 10; x++)
```

Comment:
10 - what a magic number?

the best way - add variable such as `arraySize` and use in:

int[] data1 = new int[arraySize];
            int[] data2 = new int[arraySize];
            int[] final_data = new int[arraySize];


`for (int x = 0; x < arraySize; x++)`